### DISCUSSION TITLE
Lolin 2.4" TFT discontinued - alternatives?

### DISCUSSION THREAD

**Discussion by cbb09** [HASwitchPlate/openHASP#87] [Hardware]
Hello,

I've been looking for the recommended Lolin 2.4"TFT, but it appears that the display has been discontinued and is no longer available. Are there good alternatives with integrated header for mini ESP32 or Wemos D1?

Thanks much!

**Comment by fvanroie** [comment]
> the display has been discontinued and is no longer available

That's a real shame because that form factor was very pleasant to work with. I am not aware of other displays with the same header. Depending on where you live, you might still be able to get them in a [local store](https://www.tinytronics.nl/shop/nl/display/tft/wemos-2.4-inch-tft-shield-voor-d1-mini-met-touchscreen).

The Adafruit [Featherwing TFTs](https://learn.adafruit.com/adafruit-feather/graphic-display-wings) have a *similar* socket for the ESP32 Huzzah Feather boards. Another option might be to get a ESP32 board with integrated display like the [WT32-SC01](https://lcsc.com/product-detail/Development-Boards-Development-Kits_Wireless-tag-WT32-SC01_C555472.html), [M5stack-core2](https://m5stack-store.myshopify.com/collections/m5-core/products/m5stack-core2-esp32-iot-development-kit) or [ESP32-Touchdown](https://www.tindie.com/products/dustinwattsnl/esp32-touchdown/).

If you want to DIY, there is the [FreeTouchDeck](https://github.com/DustinWatts/FreeTouchDeck) and [AZ-Touch wall enclosure](https://www.az-delivery.de/en/collections/alle-produkte/products/az-touch-wandgehauseset-mit-2-8-zoll-touchscreen-fur-esp8266-und-esp32) but I haven't tested these out myself yet. Alternately pick an [ILI9341 like these](https://haswitchplate.github.io/openHASP-docs/#displays/ILI9341_IPS/) and connect it to the ESP using a couple of jumper wires.

If you are getting the ESP's, definitely go for the ESP32 boards and ESP32-WROVER is better then ESP32-WROOM.

**Comment by fvanroie** [comment]
It seems the displays are back in stock, at least in [The Lolin shop on AliExpress](https://www.aliexpress.com/item/32919729730.html).
